### PR TITLE
Add raid mark checking to Combat Log Trigger

### PR DESCRIPTION
**Pull request by emptyrivers**

### CODE REVIEW

**Comment by emptyrivers** [comment]
wrong fork, dont mind this

File: WeakAuras/Prototypes.lua [hunk 1]
```diff
@@ -493,6 +493,36 @@ function WeakAuras.CheckCombatLogFlags(flags, flagToCheck)
...
+    COMBATLOG_OBJECT_RAIDTARGET4,
+    COMBATLOG_OBJECT_RAIDTARGET5,
+    COMBATLOG_OBJECT_RAIDTARGET6,
+    COMBATLOG_OBJECT_RAIDTARGET7,
+    COMBATLOG_OBJECT_RAIDTARGET8,
+}
+
+function WeakAuras.CheckRaidFlags(flags, flagToCheck)
+  if flagToCheck == 0 then --no raid mark
+    for mask in ipairs(marks) do
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua]: bit.band(flags, COMBATLOG_OBJECT_RAIDTARGET_MASK) == 0

File: WeakAuras/Prototypes.lua [hunk 2]
```diff
@@ -493,6 +493,36 @@ function WeakAuras.CheckCombatLogFlags(flags, flagToCheck)
...
+      if bit.band(flags,mask) then
+        return false
+      end
+    end
+    return true
+  elseif flagToCheck == 9 then --any raid mark
+    for mask in ipairs(marks) do
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua]: bit.band(flags, COMBATLOG_OBJECT_RAIDTARGET_MASK) > 0

File: WeakAuras/Prototypes.lua [hunk 3]
```diff
@@ -1233,7 +1263,17 @@ WeakAuras.event_prototypes = {
...
+        name = "sourceRaidFlags",
+        display = L["Source Raid Mark"],
+        type = "select",
+        values = "combatlog_raid_mark_check_type",
+        init = "arg",
+        store = true,
+        test = "WeakAuras.CheckRaidFlags(sourceRaidFlags,'%s')",
+        conditionType = "select",
+        conditionTest = "state and state.show and WeakAuras.CheckRaidFlags(sourceRaidFlags,'%s')",
+      },
```
Comment (InfusOnWoW) [inline WeakAuras/Prototypes.lua]: Isn't there also a destRaidFlags?

**Pull request closed without merging**